chart/files/kratos.yaml

```diff
@@ -0,0 +1,72 @@
+  courier:
+    smtp:
+      connection_uri: smtp://wrong-url
+  serve:
+    public:
+      base_url: *oryKratosURI
+      cors:
+        enabled: true
+    # admin:
+    #   base_url: http://127.0.0.1:4434/
+  selfservice:
+    default_browser_return_url: http://127.0.0.1:4455/
+    allowed_return_urls:
+      - http://127.0.0.1:4455
+    methods:
+      password:
+        enabled: true
+    flows:
+      error:
+        ui_url: http://127.0.0.1:4455/error
+      settings:
+        ui_url: http://127.0.0.1:4455/settings
+        privileged_session_max_age: 15m
+      recovery:
+        enabled: true
+        ui_url: http://127.0.0.1:4455/recovery
+      verification:
+        enabled: true
+        ui_url: http://127.0.0.1:4455/verification
+        after:
+          default_browser_return_url: http://127.0.0.1:4455/
+      logout:
+        after:
+          default_browser_return_url: http://127.0.0.1:4455/login
+      login:
+        ui_url: http://127.0.0.1:4455/login
+        lifespan: 10m
+      registration:
+        lifespan: 10m
+        ui_url: http://127.0.0.1:4455/registration
+        after:
+          password:
+            hooks:
+              -
+                hook: session
+  log:
+    level: info
+    leak_sensitive_values: false
+  identity:
+    schemas:
+      - id: default
+        # Copy config file to docker.
+        # Give file url to the config file in docker
+        url: base64://ewogICIkaWQiOiAiaHR0cHM6Ly9zY2hlbWFzLm9yeS5zaC9wcmVzZXRzL2tyYXRvcy9xdWlja3N0YXJ0L2VtYWlsLXBhc3N3b3JkL2lkZW50aXR5LnNjaGVtYS5qc29uIiwKICAiJHNjaGVtYSI6ICJodHRwOi8vanNvbi1zY2hlbWEub3JnL2RyYWZ0LTA3L3NjaGVtYSMiLAogICJ0aXRsZSI6ICJQZXJzb24iLAogICJ0eXBlIjogIm9iamVjdCIsCiAgInByb3BlcnRpZXMiOiB7CiAgICAidHJhaXRzIjogewogICAgICAidHlwZSI6ICJvYmplY3QiLAogICAgICAicHJvcGVydGllcyI6IHsKICAgICAgICAiZW1haWwiOiB7CiAgICAgICAgICAidHlwZSI6ICJzdHJpbmciLAogICAgICAgICAgImZvcm1hdCI6ICJlbWFpbCIsCiAgICAgICAgICAidGl0bGUiOiAiRS1NYWlsIiwKICAgICAgICAgICJtaW5MZW5ndGgiOiAzLAogICAgICAgICAgIm9yeS5zaC9rcmF0b3MiOiB7CiAgICAgICAgICAgICJjcmVkZW50aWFscyI6IHsKICAgICAgICAgICAgICAicGFzc3dvcmQiOiB7CiAgICAgICAgICAgICAgICAiaWRlbnRpZmllciI6IHRydWUKICAgICAgICAgICAgICB9CiAgICAgICAgICAgIH0sCiAgICAgICAgICAgICJ2ZXJpZmljYXRpb24iOiB7CiAgICAgICAgICAgICAgInZpYSI6ICJlbWFpbCIKICAgICAgICAgICAgfSwKICAgICAgICAgICAgInJlY292ZXJ5IjogewogICAgICAgICAgICAgICJ2aWEiOiAiZW1haWwiCiAgICAgICAgICAgIH0KICAgICAgICAgIH0KICAgICAgICB9LAogICAgICAgICJuYW1lIjogewogICAgICAgICAgInR5cGUiOiAib2JqZWN0IiwKICAgICAgICAgICJwcm9wZXJ0aWVzIjogewogICAgICAgICAgICAiZmlyc3QiOiB7CiAgICAgICAgICAgICAgInRpdGxlIjogIkZpcnN0IE5hbWUiLAogICAgICAgICAgICAgICJ0eXBlIjogInN0cmluZyIKICAgICAgICAgICAgfSwKICAgICAgICAgICAgImxhc3QiOiB7CiAgICAgICAgICAgICAgInRpdGxlIjogIkxhc3QgTmFtZSIsCiAgICAgICAgICAgICAgInR5cGUiOiAic3RyaW5nIgogICAgICAgICAgICB9CiAgICAgICAgICB9CiAgICAgICAgfQogICAgICB9LAogICAgICAicmVxdWlyZWQiOiBbCiAgICAgICAgImVtYWlsIgogICAgICBdLAogICAgICAiYWRkaXRpb25hbFByb3BlcnRpZXMiOiBmYWxzZQogICAgfQogIH0KfQo=
+  secrets:
+    default:
+      - lorem ipsum dolores
```

Comment:
I don't think we should be adding these defaults

Reply:
This  specific file is unused. Will delete it. Similar config is there in files/kratos-config.yaml. But it can be overwritten from values.yaml. Should I remove the secrets from files/kratos-config.yaml?